cypress/integration/editor_spec.js

```diff
@@ -18,11 +18,22 @@ context(
     it("Show a modal to rename the project", () => {
       cy.get(".sc-eJwWfJ.hZFFnD").click();
 
-      cy.get("h2").should("contain.text", "Name Your Project");
+      cy.get('[data-testid="modal"]').contains("Name Your Project", { matchCase: false });
     });
 
-    it.skip("Modal should dismiss when cancel button clicked", () => {});
+    it ("Modal should dismiss when cancel button clicked", () => {
+    
+     cy.get(".sc-eJwWfJ.hZFFnD").click();
```

Comment:
Is there a DOM element selector that could be used to more reliably reference this element?